packages/query/src/queriers/tendermint.ts

```diff
@@ -27,6 +27,7 @@ export class TendermintQuerier implements TendermintQuerierInterface {
     // the fullnode", not "wait for the tx to be included on chain.
     const req = new BroadcastTxSyncRequest({ params });
     const res = await this.client.broadcastTxSync(req);
+    console.log(res.log);
```

Comment:
Currently an issue with the effect hash: https://discord.com/channels/824484045370818580/1044457038886998137/1192940867024072795. Only seen when logged here. Relates to: https://github.com/penumbra-zone/web/issues/352

Reply:
afaict this is only present if there's something wrong - i converted it to a throw, which propagates